dev-docs/publisher-api-reference.md

```diff
@@ -1290,4 +1285,79 @@ This call returns the following DFP video ad tag URL:
+| params | object | (Required) Querystring parameters that will be used to construct the DFP video ad tag URL.  Publisher-supplied values will override values set by Prebid.js.  See below for fields. |
+| url    | string | (Optional) The video ad server URL.  When given alongside `params`, the parsed URL will be overwritten with any matching components of `params`.                                    |
+
+<a name="buildVideoUrl-options-params" />
+
+#### The `options.params` object
+
+The `options.params` object contains the parameters needed to form a URL which hits the [DFP API](https://support.google.com/dfp_premium/answer/1068325?hl=en).  It has the following fields:
+
+{: .table .table-bordered .table-striped }
+| Field             | Type   | Description                                                                                                                                                                                                                                   | Example                                         |
+|-------------------+--------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------|
+| iu                | string | (Required) DFP adUnit ID.  For more information, see the [DFP documentation on `iu`](https://support.google.com/dfp_premium/answer/1068325?hl=en#iu)                                                                                          | `/19968336/prebid_cache_video_adunit`           |
+| cust_params       | object | (Optional) Key-value pairs merged with Prebid's targeting values and sent to DFP on the video ad tag URL.  For more information, see the [DFP docs on `cust_params`](https://support.google.com/dfp_premium/answer/1068325?hl=en#cust_params) | `{section: "blog", anotherKey: "anotherValue"}` |
+| `description_url` | string | (Optional) Describes the video.  Required for Ad Exchange.  Prebid.js will build this for you unless you pass it explicitly.                                                                                                                  |                                                 |
+
+{: .alert.alert-info :}
+Note: Prebid.js will choose reasonable default values for any required DFP URL parameters that are not included in the `options.params` object.
+
+For more information about the options supported by the DFP API, see [the DFP API docs](https://support.google.com/dfp_premium/answer/1068325?hl=en#env).
+
+<a name="buildVideoUrl-usage-scenarios-and-expected-behavior" />
+
+#### Usage scenarios and expected behavior
+
+How you call `dfp.buildVideoUrl` may change depending on:
```

Comment:
I think we need to rework or possibly eliminate this section.  Options.params and Options.url are not really intended for distinct sets of use-cases.  We allow users to either pass an existing adserver URL or an object containing adserver URL querystring parameters for convenience.